### PR TITLE
onnx normalizer

### PR DESCRIPTION
**Pull request by ksaur**
Need to implement Normalizer in order for linear tests to pass (#175 )

### CODE REVIEW

File: hummingbird/ml/operator_converters/__init__.py [hunk 1]
```diff
@@ -31,4 +32,5 @@
 __pdoc__ = {}
 __pdoc__["hummingbird.operator_converters._gbdt_commons"] = True
 __pdoc__["hummingbird.operator_converters._tree_commons"] = True
+__pdoc__["hummingbird.operator_converters._linear_implementations"] = True
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/__init__.py]: I think this should be `_normalizer_implementations`,

File: tests/test_onnxml_normalizer_converter.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        # Create ONNX model by calling converter
+        onnx_model = convert(onnx_ml_model, "onnx", X)
+        # Get the predictions for the ONNX-ML model
+        session = ort.InferenceSession(onnx_ml_model.SerializeToString())
+        output_names = [session.get_outputs()[i].name for i in range(len(session.get_outputs()))]
+        onnx_ml_pred = [[] for i in range(len(output_names))]
+        inputs = {session.get_inputs()[0].name: X}
+        pred = session.run(output_names, inputs)
+        for i in range(len(output_names)):
```
Comment (interesaaat) [inline tests/test_onnxml_normalizer_converter.py]: I don't think this is actually needed. I think only one `output_name` exists. Can you please check?

File: hummingbird/ml/operator_converters/_normalizer_implementations.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+"""
+Base classes for normalizer implementations.
+"""
+
+import torch
+
+from ._base_operator import BaseOperator
+
+
+class Normalizer(BaseOperator, torch.nn.Module):
```
Comment (interesaaat) [inline hummingbird/ml/operator_converters/_normalizer_implementations.py:16]: Can you please add a one linear of documentation? Some like: Class implementing Normalizer operators in PyTorch. Supported normalizers are L1, L2 and Max.

File: hummingbird/ml/operator_converters/__init__.py (hunk 1 above)
Comment (ksaur) [reply hummingbird/ml/operator_converters/__init__.py]: whoa.  how was this working?   (the bug was from a cherrypick of my other PR, but I"m surprised this worked at alll)

Comment (ksaur) [reply hummingbird/ml/operator_converters/__init__.py]: ah yes.  just pdoc. :-P right.

**Comment by ksaur** [comment]
@interesaaat fixed, PTAL

File: tests/test_onnxml_normalizer_converter.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        onnx_ml_pred = [[] for i in range(len(output_names))]
+        inputs = {session.get_inputs()[0].name: X}
+        onnx_ml_pred = session.run(output_names, inputs)
+
+        # Get the predictions for the ONNX model
+        session = ort.InferenceSession(onnx_model.SerializeToString())
+        onnx_pred = [[] for i in range(len(output_names))]
+        onnx_pred = session.run(output_names, inputs)
+
+        return onnx_ml_pred, onnx_pred, output_names
```
Comment (interesaaat) [inline tests/test_onnxml_normalizer_converter.py]: Wait I don't see anymore where the 2 results are compared.

**Review by interesaaat — APPROVED** [review]
